### PR TITLE
Test settings

### PR DESCRIPTION
**Pull request by pirlgon**
Sleep is needed to wait for the fade effect to finish before clicking on the different buttons.

### CODE REVIEW

File: tests/test_70_settings.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+from time import sleep
+
+import pytest
+from selenium.webdriver.common.by import By
+
+
+@pytest.fixture
+def browse_settings_instance(browse_instance, browser):
```
Comment (pirlgon) [inline tests/test_70_settings.py:8]: browse_instance is a leftover

**Comment by MarionGiusti** [comment]
(Really not important but there's a typo in your first commit, missing an "e": "at th end of the tests.")

File: tests/test_25_instance_groups.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        if "prod" in tr.text
+    ][0]
+
+    assert any(
+        "Production" in group.text
+        for group in browser.select_all("#tableGroups tbody tr")
+    )
+    tr.find_element(by=By.CSS_SELECTOR, value="[data-action=delete]").click()
+    browser.select("button[type=submit]").click()
+    assert not any(
```
Comment (bersace) [inline tests/test_25_instance_groups.py:67]:
``` python
assert all(
    "Production" not in group.text
...
```

File: tests/test_25_user_groups.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+from selenium.webdriver.common.by import By
+
+
+@pytest.fixture(scope="module")
+def browse_settings_user_groups(browser_session, ui_url, admin_session):
+    browser_session.get(ui_url + "/settings/groups/role")
+
+
+@pytest.fixture(scope="module")
+def create_dev_group(browse_settings_user_groups, browser_session):
```
Comment (bersace) [inline tests/test_25_user_groups.py:11]: `group_dev` ou `user_group_dev`

File: tests/test_25_user_groups.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    browser_session.select("#buttonLoadAddGroupForm").click()
+    browser_session.select("#inputNewGroupname").send_keys("dev")
+    browser_session.select("#inputDescription").send_keys("Developers")
+    browser_session.select("button[type=submit]").click()
+
+
+def test_dev_user_group(create_dev_group, browser):
```
Comment (bersace) [inline tests/test_25_user_groups.py:18]: `test_create_user_group`

File: tests/test_99_delete_instance.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+        "ul.list-group a[href='/settings/instances']"
+    ).click()
+
+
+def test_delete_instance(browser, browse_settings_instance, agent_conf):
+    browser.select("td button.buttonDelete").click()
+    browser.select("#modalDeleteInstance button#buttonDelete").click()
+    port = agent_conf.get("temboard", "port")
+    # Check the instance has been deleted
+    assert not any(
```
Comment (bersace) [inline tests/test_99_delete_instance.py:18]:
``` python
assert all( ...
```

File: tests/test_25_instance_groups.py [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+from selenium.webdriver.common.by import By
+
+
+@pytest.fixture(scope="module")
+def browse_settings_instance_groups(browser_session, ui_url, admin_session):
+    browser_session.get(ui_url + "/settings/groups/instance")
+
+
+@pytest.fixture(scope="module")
+def create_prod_group(browse_settings_instance_groups, browser_session):
```
Comment (bersace) [inline tests/test_25_instance_groups.py:11]: `instance_group_prod` ou `group_prod`.

**Review by pgiraud — APPROVED** [review]

**Review by bersace — APPROVED** [review]

**Pull request merged**